internal/cloudapi/v2/v2_koji_test.go

```diff
@@ -477,7 +477,7 @@ func TestKojiCompose(t *testing.T) {
 				require.NoError(t, err)
 				jobTarget := osbuildJob.Targets[0].Options.(*target.KojiTargetOptions)
 				require.Equal(t, "koji.example.com", jobTarget.Server)
-				require.Equal(t, "test.img", jobTarget.Filename)
+				require.Equal(t, "test.img", osbuildJob.Targets[0].ExportFilename)
```

Comment:
I just spotted this change, which is a mistake most probably from rebasing. Will fix it...